compiler-plugin/src/main/java/io/ballerina/stdlib/persist/compiler/Constants.java

```diff
@@ -61,4 +63,19 @@ private BallerinaTimeTypes() {
         }
     }
 
+    /**
+     * Constants related to Persist datastores.
+     */
+    public static final class Datastores {
+
+        public static final String MYSQL = "mysql";
+        public static final String IN_MEMORY = "in-memory";
+        public static final String GOOGLE_SHEETS = "google-sheets";
```

Comment:
```suggestion
        public static final String IN_MEMORY = "inmemory";
        public static final String GOOGLE_SHEETS = "googlesheets";
```

Reply:
Fixed in [071d253](https://github.com/ballerina-platform/module-ballerina-persist/pull/196/commits/071d253b42c588d7d79babd70d53b07a2265cb0f)